user/contactmethod/store.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"database/sql"
 	"encoding/json"
+	"fmt"
 	"time"
 
+	n "github.com/target/goalert/notification"
```

Comment:
Don't alias packages unless it's needed -- `n` won't be obvious 200 lines down.

As a general rule, a variable/identifier's length/descriptiveness should coincide with how far away from their declaration they are used to give context. A single letter is good for something that is used on a few lines (i.e. one screen).

Also, package imports should be consistent wherever possible.